src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt

```diff
@@ -59,14 +59,26 @@ class LibraryService (
     }
   }
 
+  fun deleteGameFromLibrary(libraryId: String, gameId: String) {
```

Comment:
Do we have a separate ticket for adding tests for this?

Reply:
good point, honestly completely forgot to test this! added new test, but I also realized I never populated the LibraryService tests file?? I'll make a ticket for that